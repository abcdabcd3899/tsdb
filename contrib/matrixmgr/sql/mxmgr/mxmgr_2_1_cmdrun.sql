\c matrixmgr

SELECT matrixmgr_internal.mxmgr_cmd_run('/bin/ls', '', 'test');

SELECT matrixmgr_internal.mxmgr_cmd_run('ls', '', 'test');

SELECT matrixmgr_internal.mxmgr_cmd_run('../../bin/ls', '', 'test');

SELECT matrixmgr_internal.mxmgr_cmd_run('../bin/pg_test_timing', '--help', 'test');

SELECT matrixmgr_internal.mxmgr_cmd_run('pg_test_timing', '--help', 'test');
