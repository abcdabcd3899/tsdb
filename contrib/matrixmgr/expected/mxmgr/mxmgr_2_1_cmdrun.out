\c matrixmgr
SELECT matrixmgr_internal.mxmgr_cmd_run('/bin/ls', '', 'test');
ERROR:  absolute path not allowed
SELECT matrixmgr_internal.mxmgr_cmd_run('ls', '', 'test');
NOTICE:  (test)
 mxmgr_cmd_run 
---------------
 t
(1 row)

SELECT matrixmgr_internal.mxmgr_cmd_run('../../bin/ls', '', 'test');
ERROR:  path must be in or below the current directory
SELECT matrixmgr_internal.mxmgr_cmd_run('../bin/pg_test_timing', '--help', 'test');
ERROR:  path must be in or below the current directory
SELECT matrixmgr_internal.mxmgr_cmd_run('pg_test_timing', '--help', 'test');
NOTICE:  (test)Usage: pg_test_timing [-d DURATION]
 mxmgr_cmd_run 
---------------
 t
(1 row)

